app/services/intercom_service.rb

```diff
@@ -132,24 +111,27 @@ def self.intercom_contact_attributes(phone_number: nil, email_address: nil, clie
     attributes
   end
 
-  def self.create_new_intercom_thread(contact_id, body)
-    intercom.messages.create({ from: { type: "contact", id: contact_id }, body: body })
-  end
-
-  def self.reply_to_existing_intercom_thread(contact_id, body)
-    intercom.conversations.reply_to_last(
-      intercom_user_id: contact_id,
-      type: 'user',
-      message_type: 'comment',
-      body: body
-    )
+  def self.create_new_intercom_thread(contact, body)
+    # Read https://web.archive.org/web/20230118155416/https://forum.intercom.com/s/question/0D55c00005vHaPsCAK/create-conversation-for-lead-throws-an-error-user-not-found
```

Comment:
Bah I tested that archive.org link and it's blank. Maybe archive.is is a better choice.

https://developers.intercom.com/intercom-api-reference/reference/create-a-conversation says we should `lead` or `user`. The forum thread says we should use `user`. If we reference the forum thread, we have to change the code here.

I'll come back to this after we do our manual testing.

Reply:
archive.is links fwiw

https://archive.is/4oUOL aka https://developers.intercom.com/intercom-api-reference/reference/create-a-conversation

https://archive.is/7wTQQ aka https://forum.intercom.com/s/question/0D55c00005vHaPsCAK/create-conversation-for-lead-throws-an-error-user-not-found